omod/src/main/webapp/resources/htmlforms/ncd_other_visit.xml

```diff
@@ -257,11 +251,6 @@
                          answerConceptIds="$yesAnswer, $noAnswer" /></td>
             </tr>
 
-            <tr>
-                <th>Diagnosis</th>
-                <td><obs conceptId="$diagnosisNonCoded" /></td>
```

Comment:
Removing this is fine, but that is going to effectively hide this from use.  You have confirmed that this is not a value that anyone cares to maintain visibility into, for historical data?

Reply:
Yes this is confirmed. The field was superfluous as its intended purpose is achieved with the fields from the intake form. We noticed the issues it brought later on reviewing the data with the clinical team.